src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -334,7 +418,18 @@ void SettingsLoader::FindFragmentsAndMergeIntoUserSettings()
 
         if (std::filesystem::is_directory(path))
         {
-            parseAndLayerFragmentFiles(path, packageName);
+            // MSIX does not support machine-wide scope
+            // See https://github.com/microsoft/winget-cli/discussions/1983
+            parseAndLayerFragmentFiles(path,
+                                       packageName,
+                                       FragmentScope::User);
+
+            if (generateExtensionPackages)

```

Comment:
i see that in this case we are still _generating_ them, we are just letting them get created in `parseAndLayer` and then deleted if nobody asked for them... :(

Reply:
ah we aren't generating the objects because we didn't populate the optional. however, we are still doing package API things when the user told us not to interact w/ that source